tools/build_file_list.py

```diff
@@ -10,8 +10,8 @@
 parser.add_argument('dataset', type=str, choices=['ucf101', 'hmdb51', 'activitynet_1.2', 'activitynet_1.3'])
 parser.add_argument('frame_path', type=str, help="root directory holding the frames")
 parser.add_argument('--rgb_prefix', type=str, help="prefix of RGB frames", default='img_')
-parser.add_argument('--flow_x_prefix', type=str, help="prefix of x direction flow images", default='flow_x')
-parser.add_argument('--flow_y_prefix', type=str, help="prefix of y direction flow images", default='flow_y')
+parser.add_argument('--flow_x_prefix', type=str, help="prefix of x direction flow images", default='flow_x_')
```

Comment:
Okay I see, it doesn't matter whether `flow_x` or `flow_x_` is used since we add a `*` at the end of the filter. In order to keep consistence with the `img_` above and the default parameters in <https://github.com/yjxiong/temporal-segment-networks/blob/master/pyActionRecog/benchmark_db.py#L8>, I think `flow_x_` is more reasonable, hope to hear from you.